rocAL/rocAL_pybind/rali_pybind.cpp

```diff
@@ -95,10 +95,43 @@ namespace rali{
         return py::cast<py::none>(Py_None);
     }
 
-    py::object wrapper_label_copy(RaliContext context, py::array_t<int> array)
+    py::object wrapper_tensor32(RaliContext context, py::array_t<float> array,
+                                RaliTensorLayout tensor_format, float multiplier0,
+                                float multiplier1, float multiplier2, float offset0,
+                                float offset1, float offset2,
+                                bool reverse_channels)
     {
         auto buf = array.request();
-        int* ptr = (int*) buf.ptr;
+        float* ptr = (float*) buf.ptr;
+        // call pure C++ function
+        int status = raliCopyToOutputTensor32(context, ptr, tensor_format, multiplier0,
+                                              multiplier1, multiplier2, offset0,
+                                              offset1, offset2, reverse_channels);
+        // std::cerr<<"\n Copy failed with status :: "<<status;
+        return py::cast<py::none>(Py_None);
+    }
+
+    py::object wrapper_tensor16(RaliContext context, py::array_t<float16> array,
+                                RaliTensorLayout tensor_format, float multiplier0,
+                                float multiplier1, float multiplier2, float offset0,
+                                float offset1, float offset2,
+                                bool reverse_channels)
+    {
+        auto buf = array.request();
+        float16* ptr = (float16*) buf.ptr;
+        // call pure C++ function
+        int status = raliCopyToOutputTensor16(context, ptr, tensor_format, multiplier0,
+                                              multiplier1, multiplier2, offset0,
+                                              offset1, offset2, reverse_channels);
+        // std::cerr<<"\n Copy failed with status :: "<<status;
+        return py::cast<py::none>(Py_None);
+    }
+
+    py::object wrapper_label_copy(RaliContext context, py::object p)
+    {
+        // auto buf = array.request();
```

Comment:
remove commented lines. applies to others functions as well

Reply:
Done